doc/docs.md

```diff
@@ -439,8 +439,8 @@ println('[${int(x):-10}]') // pad with spaces on the right
 ### String operators
 
 ```v nofmt
-name := 'Bob'
-bobby := name + 'by' // + is used to concatenate strings
+nickname := 'Bob'
+name := nickname + 'by' // + is used to concatenate strings
 println(bobby) // "Bobby"
```

Comment:
please no - that makes the code invalid - the println line will fail to compile

Reply:
Besides that, both `Bob` and `Bobby` are nicknames for `Robert`.  And sometimes `Roberta`.

I would name the variables `bob` and `bobby` just to be extra explicit, then the `println` will work as-is.